docs/v1/use-cases.md

```diff
@@ -4,6 +4,13 @@ TODO: add glossary
 TODO: scrub doc post glossary
 TODO: add CF SB API as the lingua franca for blackbox services
 
+## Personas
+
+* User
+* Developer
```

Comment:
I suspect that just "Application Manager" (or Operator perhaps) might be sufficient to cover both of these.